libjob_queue/src/lsf_driver.c

```diff
@@ -859,6 +859,15 @@ static void lsf_driver_node_failure(lsf_driver_type * driver, long lsf_job_id) {
     char* hostnames = stringlist_alloc_joined_string(hosts, ", ");
     fprintf(stderr, "%s blacklisting nodes %s.\n", __func__, hostnames);
 
+    fprintf(stderr,
+            "%s Realization %ld seems to have failed as a result of LSF node failure.\n",
```

Comment:
This error message/warning is for the user and not for developers, no reason to add the leading `__func__`!